Northcoders/projects/fe-nc-news/components/Comments.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react"
+import { fetchArticleCommentsById } from "../Api"
+import { useParams } from 'react-router-dom'
+import CommentsCard from "./CommentsCard"
+
+export default function Comments () {
+    const [singleComment, setSingleComment] = useState({})
+    const [isLoading, setIsLoading] = useState(true)
+    const {article_id} = useParams()
+
+    useEffect(() => {
+        fetchArticleCommentsById(article_id)
+        .then(articleComments => {
+            setSingleComment(articleComments)
+            setIsLoading(false)
+        })
+    }, [])
+
+    if (isLoading) {
+        return (
+            <h1 className="loading">Getting comments....</h1>
+        )
+    }
+
```

Comment:
Looking good ! Nice loading pattern.  Can we render something else for an article that has no comments?